src/App.jsx

```diff
@@ -1,7 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import logo from "./logo.svg";
+import "./App.css";
 
-function App() {
+const App = () => {
   return (
```

Comment:
you can delete return here , if component do not have any logic inside the scope and only return jsx , then on nahui ne nujen 
`const App = () => <div><header></header></div>`